components/modal/styles.css

```diff
@@ -0,0 +1,44 @@
+:root {
+    --primary-color: #4a4b4b;
+    --second-color: #9a16ce;
+    --third-color: #c65efa;
+    --white: rgb(231, 227, 227);
+}
+.modal-overlay {
+    position: absolute;
+    filter: blur(2.22px) grayscale(0.46);
+    height: 100%;
+    width: 100%;
```

Comment:
insted of width:100% and height:100%,
You can use, inset: 0. 
It is a shorthand for top:0, bottom:0, right:0 and left:0

Reply:
Ok I will do that